.gitignore

```diff
@@ -1,8 +1,9 @@
-*.sublime-project
-*.sublime-workspace
-.DS_Store
-/.jekyll-metadata
-/_site/
-/benchmarks/generated
-/node_modules/
-/npm-debug.log
```

Comment:
Sorry, didn't mean to commit that. I'm new to the whole pull requests thing so I'm making some mistakes.